### DISCUSSION TITLE
Is there a way to set max threads for specific test?

### DISCUSSION THREAD

**Discussion by omer-za** [xunit/xunit#2237] [Question] [answered]
I want to have all tests running by default in parallel while specific tests to run only 1 by 1.
When using Collection they are just not running in parallel to each other colllection but there's still other tests that are running

**Comment by bradwilson** [comment, marked as answer]
Yes, you can do that with `DisableParallelization` on the [`[CollectionDefinition]`](https://github.com/xunit/xunit/blob/v2/src/xunit.core/CollectionDefinitionAttribute.cs) attribute for your collection. Setting this to `true` means that any test in that collection will not run in parallel with any other collections; you effectively get a serialized run of all non-parallel collections after all the parallel-compatible collections have finished.

See the "Changing Default Behavior" sub-section of the ["Parallelism in Test Frameworks" section of "Running Tests in Parallel"](https://xunit.net/docs/running-tests-in-parallel#parallelism-in-test-frameworks) in the documentation.
